src/Prod/README.md

```diff
@@ -0,0 +1,152 @@
+# Prod
+```haskell
+Prod Number
+```
+`Prod` is a `Monoid` that will combine two numbers using Multiplication.
+
+```js
+const Prod = require('crocks/Prod')
+const compose = require('crocks/helpers/compose')
+const concat = require('crocks/pointfree/concat')
+
+const emptyProd = Prod.empty()
+const tenProd = Prod(10)
+const hundredProd = Prod(100)
+
+emptyProd
+  .concat(emptyProd)
+//=> Prod 1
+
+tenProd
+  .concat(hundredProd)
+//=> Prod 1000
+
+// prodBy2 :: Number -> Prod
+const double =
+  compose(concat(Prod(2)), Prod)
+
+double(4)
+//=> Prod 8
+
+double(11)
+//=> Prod 22
+```
+
+## Implements
+`Semigroup`, `Monoid`
+
+## Constructor Methods
+
+### empty
+```haskell
+Prod.empty :: () -> Prod
+```
+
+`empty` provides the identity for the `Monoid` in that when the value it
+provides is `concat`ed to object other value, it will return the other value.
+In the case of `Prod` the result of `empty` is `1`. `empty` is available on
+both the Constructor and the Instance for convenience.
+```js
+const Prod = require('crocks/Prod')
+
+Prod.empty()
+//=> Prod 1
+
+Prod.empty()
+  .concat(Prod.empty())
+//=> Prod 1
+
+Prod(4)
+  .concat(Prod.empty())
+//=> Prod 4
+```
+
+
+### type
+```haskell
+Prod.type :: () -> String
+```
+
+`type` provides a string representation of the type name for a given type in
+`crocks`. While it is used mostly internally for law validation, it can be
+useful to the end user for debugging and building out custom types based on
+the standard `crocks` types. While type comparisons can easily be done manually
+by calling `type` on a given type, using the `isSameType` function hides much
+of the boilerplate. `type` is available on both the Constructor and the
+Instance for convenience.
+
+```js
+const Prod = require('crocks/Prod')
+const Sum = require('crocks/Sum')
+const isSameType = require('crocks/predicates/isSameType')
+
+const prod5 = Prod(5)
+
+prod5.type()
+//=> "Prod"
+
+isSameType(Sum, prod5)
+//=> false
+
+isSameType(Prod, prod5)
+//=> true
+```
+
+## Instance Methods
+
+### concat
+```haskell
+Prod ~> Prod -> Prod
+```
+
+`concat` is used to combine (2) `Semigroup`s of the same type under an
+operation specified by the `Semigroup`. In the case of `Prod`, it will multiply
```

Comment:
> operation specified by the `Semigroup`. In the case of `Prod`, it will multiply the two `Number`s

Should read more like:
operation specified by the `Semigroup`. In the case of `Prod`, `concat` will multiply the (2) `Number`s